### PR TITLE
Initial SQL style guide.

### PR DESCRIPTION
**Pull request by RapidOwl**

### CODE REVIEW

File: StyleGuide.sql [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+-- Schema = snake_case
+-- Table = snake_case
+-- Field = snake_case
+
+-- Every statement should end with a semi-colon
+SELECT
+	field_one,
+	field_two
+FROM
+	schema_name.table_name
```
Comment (Nevett) [inline StyleGuide.sql]: Should there be a semicolon here?

Comment (RapidOwl) [reply StyleGuide.sql]: Hah. Yes. There should be.

File: StyleGuide.sql [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+-- Capitalise keywords and function names
+-- Qualify table names with schema name
+-- Each field and part of statement should be on its own line
+-- Indent for readability
+UPDATE
+	[schema].[Table]
+SET
+	[FieldOne] = 'Thing',
+	[FieldTwo] = GETDATE()
```
Comment (caterpaula) [inline StyleGuide.sql]: comma should be at the beginning of next line, causes smaller diffs when new fields are added (1 line changed compared to 2)

File: StyleGuide.sql [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+-- Casing:
+-- Database name = UpperCamelCase
+-- Schema = lower case
+-- Table = UpperCamelCase
+-- Field = UpperCamelCase
+
+-- Names (field, table, schema) should be wrapped in []
+SELECT
+	[FieldOne],
+	[FieldTwo]
```
Comment (caterpaula) [inline StyleGuide.sql]: comma should be at the beginning of next line, causes smaller diffs when new fields are added (1 line changed compared to 2)

File: StyleGuide.sql [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+-- Casing:
+-- Database name = snake_case
+-- Schema = snake_case
+-- Table = snake_case
+-- Field = snake_case
+
+-- Every statement should end with a semi-colon
+SELECT
+	field_one,
+	field_two
```
Comment (caterpaula) [inline StyleGuide.sql]: comma should be at the beginning of next line, causes smaller diffs when new fields are added (1 line changed compared to 2)

File: StyleGuide.sql (hunk 2 above)
Comment (RapidOwl) [reply StyleGuide.sql]: Done.

File: StyleGuide.sql (hunk 3 above)
Comment (RapidOwl) [reply StyleGuide.sql]: Done.

File: StyleGuide.sql (hunk 4 above)
Comment (RapidOwl) [reply StyleGuide.sql]: Done.

**Pull request merged**